packages/schema/src/config/common.ts

```diff
@@ -355,11 +355,14 @@ export default defineUntypedSchema({
       '**/*.stories.{js,ts,jsx,tsx}', // ignore storybook files
       '**/*.{spec,test}.{js,ts,jsx,tsx}', // ignore tests
       '**/*.d.ts', // ignore type declarations
-      '.output',
-      '.git',
-      '.cache',
-      await get('analyzeDir'),
-      await get('buildDir'),
+      '**/dist',
+      '**/.vercel',
+      '**/.netlify',
+      '**/.output',
+      '**/.git',
+      '**/.cache',
```

Comment:
We can possibly read these from `.gitignore` to be extendable. `.data` also can/should be added to the list coming from next version of nitro for `data:` storage.

Reply:
.gitignore seems a nice/better idea but I wonder if someone has (for example) a generated schema that they want to be in build but not committed to git...

Maybe worth including in 3.7 with explanation on how to 'unignore' it?